### PR TITLE
Fix #2333: Prevent crash on Administrator Controls when it is selected again in navigation drawer

### LINKED ISSUE #2333: Click on Admin Controls crash in Admin Controls
**Describe the bug**
If we are in `Administrator Controls` screen and select `Administrator Controls` from Navigation Drawer again, it crashes the app.

**To Reproduce**
1. Select `Admin` profile.
2. Open `Navigation Drawer`
3. Select `Administrator Controls`
4. Open `Navigation Drawer`
5. Select `Administrator Controls`
The application will crash.

**Expected behavior**
The application should not crash and the navigation drawer should be closed.

**Screenshot**
Screenshot 2020-12-29 at 11 26 48 AM

### PR DESCRIPTION
**Pull request by ShridharGoel**
## Explanation
Fixes #2333 

## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by anandwana001** [comment]
Looks like duplicate - https://github.com/oppia/oppia-android/pull/1867
If this approach looks good, we can go with this one. @rt4914

**Comment by ShridharGoel** [comment]
@anandwana001 Thanks for informing. Did not notice that a PR already exists for this, since a new issue was created - #2333.

File: app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt [hunk 1]
```diff
@@ -100,6 +100,11 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
         getHeaderViewModel().profile.set(it)
         getFooterViewModel().isAdmin.set(it.isAdmin)
         binding.administratorControlsLinearLayout.setOnClickListener {
+          if (getFooterViewModel().isAdministratorControlsSelected.get()!!) {
```
Comment (rt4914) [inline app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt]: Avoid using `!!`. Instead you can write it as 
`if (getFooterViewModel().isAdministratorControlsSelected.get() == true) {`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ShridharGoel Suggested nit change.

@anandwana001 This approach does look good.

File: app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt (hunk 1 above)
Comment (ShridharGoel) [reply app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt]: @rt4914 Thanks for the suggestion, updated.

**Comment by anandwana001** [comment]
closing this PR due to force push.

**Pull request closed without merging**